src/index.js

```diff
@@ -1,17 +1,42 @@
-import _ from 'lodash';
-import printMe from './print.js';
+import './style.css';
 
-function component() {
-  const element = document.createElement('div');
-  const btn = document.createElement('button');
-
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+const toDoTasks = [
+  { index: 1, description: 'wash dishes', completed: false },
+  { index: 2, description: 'complete To Do List Project', completed: false },
+  { index: 3, description: 'Fix car', completed: false },
+];
 
-  btn.innerHTML = 'Click me and check the console!';
-  btn.onclick = printMe;
+let todoMarkup = `      
+<ul class="todo-ul">
+<li class="todo-li">
+  <h1>Today's To Do</h1>
+  <p><i class="fas fa-sync-alt"></i></p>
+</li>
+<li class="todo-li">
+  <input
+    type="text"
+    class="todo-input"
+    placeholder="Add to your list..."
+  />
+</li>
+</ul>`;
 
-  element.appendChild(btn);
+const clearButton = "<li><button class='clear-btn'>Clear All completed</button></li>";
 
+function component() {
+  const element = document.createElement('div');
+  element.classList.add('todo-div');
+  toDoTasks.forEach((task) => {
+    todoMarkup += `<li class="todo-li">
+    <div class="todo-input-div">
+      <input type="checkbox" />
+      <p class='todo-description'> ${task.description}</p>
+    </div>
+     
+    <p><i class="fas fa-ellipsis-v"></i></p>
+  </li>`;
+  });
+  element.innerHTML = todoMarkup + clearButton;
   return element;
 }
```

Comment:
- [ ] It would be so much better if the tasks are appear in order of the index values. As is one of the project requirements. Please fix this minor issue. 
🛠️ 💯